views/layout.jade

```diff
@@ -9,6 +9,15 @@ html
   body.bg-dark
     nav.clearfix.bg-light.white.sm-show
       div.container
+        //- if messages.error
```

Comment:
It looks like in some cases we're exposing the message directly and in others we're setting the message with a descriptive key on a `messages` object -- I think the `messages` object is probably preferable, but we'll need to make sure to initialize it on every request. Mostly I'd like to get these lines uncommented -- could you add the initialization of `messages` (we can just set it on `req` when the request comes in, I think, but open to suggestions) and uncomment these lines?